ksp_plugin/pile_up.hpp

```diff
@@ -30,10 +36,41 @@ class PileUp final {
  public:
   explicit PileUp(std::list<not_null<Vessel*>>&& vessels);
 
+  void set_mass_and_intrinsic_force(
```

Comment:
I would have a preference for two setters, coupling mass and force seems unnecessary.